docs/rootless.md

```diff
@@ -0,0 +1,47 @@
+# Rootless mode (Experimental)
+
+Requirements:
+- runc (May 30, 2018) or later
```

Comment:
The upstream master is broken as of writing due to a merge conflict but will be fixed immediately in  https://github.com/opencontainers/runc/pull/1808